### ISSUE TITLE
docker_container does not allow to disable container logging via log_driver

### ISSUE THREAD

**Issue by tumbl3w33d** [ansible/ansible#18984]
_From @tumbl3w33d on October 20, 2016 20:2_

##### ISSUE TYPE
- Bug Report
##### COMPONENT NAME

docker_container
##### ANSIBLE VERSION

```
ansible 2.2.0 (devel 28feba2fb3) last updated 2016/09/14 12:25:32 (GMT +200)
  lib/ansible/modules/core: (detached HEAD ae6992bf8c) last updated 2016/09/14 12:25:32 (GMT +200)
  lib/ansible/modules/extras: (detached HEAD afd0b23836) last updated 2016/09/14 12:25:32 (GMT +200)
```
##### CONFIGURATION

N/A
##### OS / ENVIRONMENT

N/A
##### SUMMARY

It is not possible to disable container logging, although 'none' is a valid option for docker run.

> none  Disables any logging for the container. docker logs won’t be available with this driver.
> Source: https://docs.docker.com/engine/admin/logging/overview/
##### STEPS TO REPRODUCE

```
  docker_container:
   [...]
    log_driver: none
```
##### EXPECTED RESULTS

Container with disabled logging started.
##### ACTUAL RESULTS

```
FAILED! => {"changed": false, "failed": true, "msg": "value of log_driver must be one of: json-file,syslog,journald,gelf,fluentd,awslogs,splunk, got: none"}
```

_Copied from original issue: ansible/ansible-modules-core#5337_

**Comment by tumbl3w33d** [comment]
_From @ansibot on October 20, 2016 20:4_



@ansible ping, this issue is waiting for your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by tumbl3w33d** [comment]
_From @ansibot on November 5, 2016 8:25_

@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by tumbl3w33d** [comment]
_From @ansibot on November 22, 2016 9:43_

@ansible, ping. This issue is still waiting on your response.
[click here for bot help](https://github.com/ansible/ansibullbot/blob/master/ISSUE_HELP.md)

**Comment by tumbl3w33d** [comment]
_From @ansibot on December 7, 2016 19:51_

This repository has been locked. All new issues and pull requests should be filed in https://github.com/ansible/ansible

Please read through the <a href="http://docs.ansible.com/ansible/dev_guide/repomerge.html#move-issues-and-prs-to-new-repo">repomerge</a> page in the dev guide. The guide contains links to tools which automatically move your issue or pull request to the ansible/ansible repo.

**Comment by tumbl3w33d** [comment]
I consider this fixed by cfed617dba3c66d252cd25843ef5e3855f4c1edc so it can be closed.
